src/main/java/com/example/demo/model/File.java

```diff
@@ -0,0 +1,46 @@
+package com.example.demo.model;
+
+public class File {
```

Comment:
Try to find a better name for your internal file's representation that doesn't clash with the JDK's `java.io.File`. Otherwise it may confuse other developers